gsy_framework/data_classes.py

```diff
@@ -350,23 +352,27 @@ def __init__(self, id: str, creation_time: DateTime, offer_bid: Union[Offer, Bid
         self.buyer_origin_id = buyer_origin_id
         self.seller_id = seller_id
         self.buyer_id = buyer_id
+        self.matching_requirements = matching_requirements
 
     def __str__(self) -> str:
         return (
             f"{{{self.id!s:.6s}}} [origin: {self.seller_origin} -> {self.buyer_origin}] "
             f"[{self.seller} -> {self.buyer}] {self.traded_energy} kWh @ {self.trade_price}"
             f" {round(self.trade_rate, 8)} "
-            f"{self.offer_bid.id} [fee: {self.fee_price} cts.]")
+            f"{self.offer_bid.id} [fee: {self.fee_price} cts.] "
+            f"{self.matching_requirements if self.matching_requirements else ''}")
```

Comment:
Minor, but can be changed to:
```suggestion
            f"{self.matching_requirements or ''}")
```